src/App.jsx

```diff
@@ -1,107 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+const posts = postsFromServer.map(post => ({
+  ...post,
+  user: getUserbyId(post.userId),
+  comments: getCommentsbyId(post.id),
+}));
```

Comment:
You can use destructuring as well
```suggestion
const posts = postsFromServer.map(({ userId, id }) => ({
  ...post,
  user: getUserbyId(userId),
  comments: getCommentsbyId(id),
}));
```